prime-router/metadata/schemas/WATERS/waters-covid-19.schema

```diff
@@ -30,7 +30,7 @@ elements:
   # I added this and made it required so we remember to discuss.
   # Very important field for routing data to states.
   - name: ordering_facility_state
-    cardinality: ONE
+    cardinality: ZERO_OR_ONE
     csvFields: [{ name: ordering_facility_state}]
```

Comment:
per discussion with the waters team.  They don't currently have ordering_facility_state; some states require this, in order to filter properly.